gems/pending/spec/postgres_ha_admin/failover_databases_spec.rb

```diff
@@ -1,32 +1,48 @@
-describe FailoverDatabases do
+require 'postgres_ha_admin/failover_databases'
+require 'postgres_ha_admin/postgres_ha_admin'
+require 'spec/support/test_env_helper'
```

Comment:
Ah I see. In that case see https://github.com/ManageIQ/manageiq/pull/10103/files#r72844176